patches/minecraft/net/minecraft/world/level/block/entity/BlockEntityType.java.patch

```diff
@@ -0,0 +1,29 @@
+--- a/net/minecraft/world/level/block/entity/BlockEntityType.java
++++ b/net/minecraft/world/level/block/entity/BlockEntityType.java
+@@ -219,7 +_,7 @@
+     public static final BlockEntityType<TrialSpawnerBlockEntity> TRIAL_SPAWNER = register("trial_spawner", TrialSpawnerBlockEntity::new, Blocks.TRIAL_SPAWNER);
+     public static final BlockEntityType<VaultBlockEntity> VAULT = register("vault", VaultBlockEntity::new, Blocks.VAULT);
+     private final BlockEntityType.BlockEntitySupplier<? extends T> factory;
+-    private final Set<Block> validBlocks;
++    private final net.minecraftforge.common.util.ValidBlocksSet validBlocks;
```

Comment:
This is heavily reflected/edited by mods so its not good to change this field type.